app/controllers/hub/state_file/efile_submissions_controller.rb

```diff
@@ -48,6 +48,11 @@ def state_counts
 
       private
 
+      def error_redirect
+        flash[:alert] = "There was a problem generating the tax return pdf."
+        redirect_back(fallback_location: request.referer)
+      end
+
```

Comment:
is this method used in the hub somewhere?

Reply:
Yes this is used in the `show_pdf` method above